components/OpenAPIComparison.js

```diff
@@ -15,41 +15,41 @@ export default function OpenAPIComparison({ className = '' }) {
     });
 
     return (
-        <div className={`${className} flex flex-wrap text-center`}>
+        <div className={ `${ className } flex flex-wrap text-center` }>
             <div className="flex-1 border border-black p-2 mr-1">
                 <h3 className="text-lg font-sans font-medium mb-4 ml-2">OpenAPI 3.0</h3>
 
                 <div>
-                    <div className={(hoverState.Info ? `bg-blue-100 ` : ` `) + `border border-blue-300 p-2 m-2`} onMouseOver={() => setHoverState(prevState => ({ ...prevState, Info: true }))} onMouseLeave={() => setHoverState({ ...hoverState, Info: false })}>
+                    <div className={ (hoverState.Info ? `bg-blue-100 ` : ` `) + `border border-blue-300 p-2 m-2` } onMouseOver={ () => setHoverState(prevState => ({ ...prevState, Info: true })) } onMouseLeave={ () => setHoverState({ ...hoverState, Info: false }) } data-testid="OpenAPI-sec1">
```

Comment:
```suggestion
                    <div className={ (hoverState.Info ? `bg-blue-100 ` : ` `) + `border border-blue-300 p-2 m-2` } onMouseOver={ () => setHoverState(prevState => ({ ...prevState, Info: true })) } onMouseLeave={ () => setHoverState({ ...hoverState, Info: false }) } data-testid="OpenAPI-sec-info">
```

Don't make the test ids as 1,2,3..., use the correct names to identify them easily.

Applicable everywhere.

Reply:
^ @reachaadrika